Auth0/CredentialsManager.swift

```diff
@@ -41,9 +41,10 @@ public struct CredentialsManager {
     /// - Parameters:
     ///   - authentication: Auth0 authentication instance
     ///   - storeKey: Key used to store user credentials in the keychain, defaults to "credentials"
```

Comment:
Please update Doc Header with new param.

Reply:
Documentation for the new argument is now provided in the latest commit.